### PR TITLE
MacOS install instructions

### PR DESCRIPTION
**Pull request by Gazook89**
I went through the process of getting Homebrewery working on MacOS *without* using Docker and documented my effort, which is now this readme.md.  Some of the steps I already had in place such as installation of git and node, so I haven't been able to test all the steps as I've written them, but I'm pretty confident it is correct.  Until the whole process is confirmed by one or more people though, this is a draft.  

It is rather wordy compared to the main repo readme, but I wrote it with much more depth for near-beginners.  Mostly this was for my benefit as a near-beginner myself so I can go back and see what I did in case I need to.

I am *very* happy to get pointers to make all of this easier.

@Gazook89 To complete before merging:

- [ ] Add a link in the main readme.md
- [ ] Remove Step 8 (V3 is now enabled in the repo by default)
- [ ] Apply other reviewer comments

### CODE REVIEW

File: README.macOS.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
+# Local Install on MacOS with MongoDB
+This will walk through the steps to getting Homebrewery running locally on your computer without needing internet access to create and save brews.  It provides steps on how to get set up with either a **production** environment (if you just want to make brews) or a **development** environment (if you want to submit code changes to project).  Finally, it will provide instruction on how to update to the lastest release over time.  This will be written with no assumption of technical skill as I have very little myself.
+
+## Step 1: Install [Homebrew](https://brew.sh)
+Homebrew is a *package manager* for macOS and Linux, and we will use it to install NodeJS, MongoDB, and git.  You likely will need to be an Administrator on your machine.
```
Comment (calculuschild) [inline README.macOS.md:5]: Can you add a couple words to just clarify that Homebrew is not the same as the Homebrewery? Somebody is going to get confused I'm sure.

File: README.macOS.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+	"host" : "homebrewery.local.naturalcrit.com:8000",
+	"naturalcrit_url" : "local.naturalcrit.com:8010",
+	"secret" : "secret",
+	"web_port" : 8000,
+	"enable_v3" : true
+}
+```
+
+_*[skip_worktree](https://compiledsuccessfully.dev/git-skip-worktree/) tells git to ignore a folder or file, similar to .gitignore, except it works on files that are already tracked and .gitignore does not._
+
```
Comment (calculuschild) [inline README.macOS.md:105]: I think v3 is now in the config file.

**Review by calculuschild — CHANGES_REQUESTED** [review]
Just a couple things. Sorry this has taken so long to review.

I'm just going to trust you on most of the Mac-specific stuff because I don't have a way to verify it.

**Comment by calculuschild** [comment]
@Gazook89 Do you think we can finish this one off? It's been sitting for a while. Or perhaps @G-Ambatte if you want to contribute to this as part of #1904

File: README.macOS.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+```
+- In the Terminal, run `source ~/.zshrc`
+- Now you should be able to run the shortcuts.  In the Terminal, run `brew services status` and after hitting Enter you should get a list of services running and their status (if you started your MongoDB service it should be listed).
+
+## Step 4: Create Homebrewery Database
+In the Terminal:
+- type `mongosh` and you will get a result indicating you've opened Mongo
+- type `use homebrewery` to create a database for Homebrewery
+- type `db.brews.insert({"title":"test"})` and you will insert a new document titled "test" into the "brews" collection, in the Homebrewery database.
+- type db.brews.find().pretty() to display a list of the documents in the "brews" collection...you should get one result back, the document you just created.
```
Comment (Gazook89) [inline README.macOS.md:55]: this should be `db.brews.find().pretty()`  (just put that in a code block).

**Comment by calculuschild** [comment]
@Gazook89 I see you updated this some more. Any chance we can address my remaining comments and merge this?

**Comment by Gazook89** [comment]
So i haven't been able to get HB working locally again (without using Docker, which I haven't tried since I had it installed not with docker).   I'm not sure if I've just screwed up the environment too much, or something changed in HB that I can't figure out, but either way the instructions above haven't been able to help me with the reinstall.  Additionally, I think there are steps in this that are superfluous, but I can't figure out exactly which ones.

Until I am able to learn more about the underlying issues I can't really vouch for this PR as-is, so I suggest that it is closed as unfinished for now.  I have been working on other projects solely to learn javascript lately, and soon-ish will start trying to learn backend stuff and thus more about environments etc.....so someday I'll maybe have a good enough grip to come back to this PR(?)

**Comment by calculuschild** [comment]
Sounds good. I'll close it for now then.

File: README.macOS.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+- add the following lines to the file and save it.
+```
+alias mongod-run="brew services run mongodb-community@5.0"
+alias mongod-start="brew services start mongodb-community@5.0"
+alias mongod-status="brew services list"
+alias mongod-stop="brew services stop mongodb-community@5.0"
+alias mongod-restart="brew services restart mongodb-community@5.0"
+```
+- In the Terminal, run `source ~/.zshrc`
+- Now you should be able to run the shortcuts.  In the Terminal, run `brew services status` and after hitting Enter you should get a list of services running and their status (if you started your MongoDB service it should be listed).
```
[the remark this answers is not in the document]
Comment (Gazook89) [reply README.macOS.md:48]: confirmed the above comment is true as i follow my own install instructions again.

File: README.macOS.md (hunk 3 above)
Comment (Gazook89) [reply README.macOS.md:55]: line 54 gets this error:
```zsh
This gets an error about deprecation:
```zsh
homebrewery> db.brews.insert({"title":"test"})
DeprecationWarning: Collection.insert() is deprecated. Use insertOne, insertMany, or bulkWrite.
```

**Pull request closed without merging**